jest.config.js

```diff
@@ -2,4 +2,4 @@
 module.exports = {
   preset: 'ts-jest',
   testEnvironment: 'node',
-};
\ No newline at end of file
+};
```

Comment:
can you undo this change? I wanna see CI fail :)

Reply:
Sure. Done

Reply:
great, good stuff, you can put it back to fix CI

Reply:
Done